README.md

```diff
@@ -1,3 +1,54 @@
 # http4ts
+**Server as a Function http toolkit for Typescript**
 
 [![Actions Status](https://github.com/http4ts/http4ts/workflows/Node%20CI/badge.svg)](https://github.com/http4ts/http4ts/actions)
+
+http4ts is a minimal http library for Javascript environments (nodejs, deno etc.) implementing the pattern of server as a function. In http4ts a server is just a function with the following signature:
+``` ts
+type HttpHandler = (req: HttpRequest) => HttpResponse | Promise<HttpResponse>;
+```
+A simple server application can be found in [examples directory](https://github.com/http4ts/http4ts/tree/master/src/examples).
```

Comment:
I think that it is better to place an example directly here. Probably more than 1 which also includes filters. Usually users don't dig very deep if they are not sold in the first sight.